cl/lib/juriscraper_utils.py

```diff
@@ -0,0 +1,22 @@
+import pkgutil
+import juriscraper
+import importlib
+
+
+def get_scraper_object_by_name(court_id):
+    """Identify and instantiate a Site() object given the name of a court
+
+    :param court_id: The ID of a site; must correspond with the name of a
+    module without the underscore suffix.
+    :type court_id: str
+    :return: An instantiated Site() object from the module requested
+    :rtype: juriscraper.AbstractSite.Site
```

Comment:
I'd mention here that it could return None if your import failed. Or if we want to be even more defensive, we could make it throw an exception in that case, but that's fairly small.